src/realm/object-store/sync/app.cpp

```diff
@@ -264,7 +264,11 @@ void App::UserAPIKeyProviderClient::create_api_key(
     };
 
     nlohmann::json body = {{"name", name}};
-    Request req = {.method = HttpMethod::post, .url = route, .body = body.dump(), .uses_refresh_token = true};
+    Request req;
```

Comment:
Sorry I didn't notice this sooner, but all of these should either do `Request req = {};` or explicitly set `req.timeout_ms = 0`. Or even better, modify https://github.com/realm/realm-core/blob/67a72373d8945b427dd6fcd606f4b9a1f445942b/src/realm/object-store/sync/generic_network_transport.hpp#L174-L203 to add default values for `method = HttpMethod::get`, `timeout_ms = 0`, `uses_refresh_token = false` (I don't know that these are the best defaults, but they are currently the defaults used if you use `Request{...}`)

Reply:
I added default values. I will apply this change in realm-object-store repo as well.